volestipy/volestipy/src/bindings.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <math.h>
+#include "bindings.h"
+
+using namespace std;
+
+
+HPolytopeCPP::HPolytopeCPP() {}
+HPolytopeCPP::HPolytopeCPP(double *polytope, double *bias, int n_hyperplanes, int n_variables) {
```

Comment:
you call `polytope` but this is a matrix copied to the matrix A, I found this confusing

Reply:
Fixed this:
polytope ->A_np
bias -> b_np